packages/wifi_iot/ios/Classes/SwiftWifiIotPlugin.swift

```diff
@@ -113,8 +113,9 @@ public class SwiftWifiIotPlugin: NSObject, FlutterPlugin {
                     // trigger access for local network
                     triggerLocalNetworkPrivacyAlert();
                 }
+            }else{
```

Comment:
Very tiny nit, add spaces between the `else` keyword. Other than that, looks good to me.